pkg/prometheus/promcfg.go

```diff
@@ -537,6 +555,9 @@ func (cg *configGenerator) generatePodMonitorConfig(version semver.Version,
 		for _, c := range ep.RelabelConfigs {
 			relabelings = append(relabelings, generateRelabelConfig(c))
 		}
+		// Because of security risks, whenever enforcedNamespaceLabel is set, we want to append it to the
```

Comment:
can we (shortly) summarize what those security risks are? just to answer the question what threat model we protect against.

Reply:
It already says that it ensures it overrides any other relabeling. What did you have in mind to add?

Reply:
if that is all, then this is fine.